packages/ui-components/src/components/labels-dropdown-input-item/labels-dropdown-input-item.tsx

```diff
@@ -0,0 +1,33 @@
+import { Component, Event, EventEmitter, Prop, h } from '@stencil/core';
+import { LabelsInputItem, LabelsInputItemEvents } from './labels-dropdown-input-item.types';
+import { EIconName } from '../icon/icon.types';
+
+@Component({
+	tag: 'kv-labels-dropdown-input-item',
+	styleUrl: 'labels-dropdown-input-item.scss',
+	shadow: true
+})
+export class KvLabelsDropdownInputItem implements LabelsInputItem, LabelsInputItemEvents {
+	/** @inheritdoc */
+	@Prop({ reflect: true }) value!: string;
+	/** @inheritdoc */
+	@Prop({ reflect: true }) label!: string;
+
+	/** @inheritdoc */
+	@Event() removeClick: EventEmitter<string>;
+
+	render() {
+		return (
+			<div class="labels-input-item" onClick={e => e.stopPropagation()}>
```

Comment:
You could use a function to handle the icon click and also add the stop the propagation there. Unless you are doing this to reduce code